textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,21 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+            <classifier>models</classifier>
+        </dependency>
+
+
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
How about this: I'll keep the current test cases for this PR. For my next PR to allow Part of speech extraction, I'll use Mockito instead of the ScanBasedSourceOperator?